tests/init/neg/super-resolution2.scala

```diff
@@ -0,0 +1,19 @@
+abstract class A:
+  val n: Int
+  def foo(): Int = n
+
+trait B:
+  val m: Int
+  def foo(): Int = m
+
+class N extends A with B:
+  override def foo(): Int = 10 * super.foo()
```

Comment:
Should this test have an additional field to ensure that this `def foo` is not considered to be called during initialization (like the `n` field in the other test)?